src/components/ftbot/TradeListNav.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <b-list-group>
+    <b-list-group-item
+      v-for="trade in trades.sort((a,b) => a.open_timestamp < b.open_timestamp)"
```

Comment:
You can't really mutate props - (trade.sort will mutate the prop) - so CI will freak at that (among others ...).

most likely, a `trade.splice().sort()` should work - but i'd avoid doing this in the template - but move the logic to the setup function instead.

Reply:
Ah that's right, forgot sort doesn't return a new array, will fix it.